src/components/quickStarts.stories.mdx

```diff
@@ -484,4 +426,216 @@ will automatically stop any video processor that was previously running.
+    };
+
+    try {
+      await meetingManager.join(joinData);
+    } catch {
+      console.error('Something went wrong');
+    }
+  };
+
+  <button onClick={joinMeeting}>Join Meeting</button>;
+};
+
+const App = () => (
+  <MeetingProvider>
+    <MyApp />
+  </MeetingProvider>
+);
+```
+
+## Receive Amazon Chime SDK Meeting Events
+
+You can receive Amazon Chime SDK meeting events in your application by using the `MeetingProvider` and the `useMeetingEvent` hook.
+For more information, please check the documentation on the following components:
+- [MeetingEventProvider](/story/sdk-providers-meetingeventprovider--page)
+- [useMeetingEvent](/story/sdk-hooks-usemeetingevent--page)
+
+```jsx
+import React from 'react';
+import { MeetingProvider, useMeetingEvent } from 'amazon-chime-sdk-component-library-react';
+
+const MyApp = () => (
+  <MeetingProvider>
+    <MeetingEventReceiver />
+  </MeetingProvider>
+);
+
+const MeetingEventReceiver = () => {
+  const meetingEvent = useMeetingEvent();
+  console.log('Received a meeting event', meetingEvent);
+  return null;
+};
+```
+
+## Send and Receive Data Messages
+
+The Data Messages feature allows attendees to send small data messages (no larger than 2KB) to other attendees. It can be used to notify attendees of changes in the meeting state or to develop custom collaboration features. Each message is sent on a particular topic, which allows you to tag messages according to their function to make it easier to handle messages of different types.
+
+To learn more, check [Amazon Data Message](https://aws.amazon.com/about-aws/whats-new/2020/05/amazon-chime-sdk-data-messages-real-time-signaling/).
+
+There are two corresponding APIs: [`realtimeSubscribeToReceiveDataMessage`](https://aws.github.io/amazon-chime-sdk-js/interfaces/audiovideofacade.html#realtimesubscribetoreceivedatamessage) and [`realtimeUnsubscribeFromReceiveDataMessage`](https://aws.github.io/amazon-chime-sdk-js/interfaces/audiovideofacade.html#realtimeunsubscribefromreceivedatamessage). To use them, you could use either [`useMeetingManager`](https://aws.github.io/amazon-chime-sdk-component-library-react/?path=/story/sdk-hooks-usemeetingmanager--page) hook or [`useAudioVideo`](https://aws.github.io/amazon-chime-sdk-component-library-react/?path=/story/sdk-hooks-useaudiovideo--page) hook.
+
+```ts
+const audioVideo = useAudioVideo();
+
+if (audioVideo) {
+  audioVideo.realtimeSubscribeToReceiveDataMessage(topic: string, callback: (dataMessage: DataMessage) => void): void;
+  audioVideo.realtimeUnsubscribeFromReceiveDataMessage(topic: string): void
+}
+```
+
+```ts
+const meetingManager = useMeetingManager();
+
+meetingManager.audioVideo?.realtimeSubscribeToReceiveDataMessage(topic: string, callback: (dataMessage: DataMessage) => void): void
+meetingManager.audioVideo?.realtimeUnsubscribeFromReceiveDataMessage(topic: string): void
+```
+
+For example, the remote control function to mute/unmute all other participants using data message.
```

Comment:
Why do we want to add this example? Its kinda an unsecured way to implement remote mute so not sure if we want to use this as an example :)

Reply:
Changed the code example to sending a simple message